File: dart/lib/poco.dart
class Vector4 {
  double x = 0;
  double y = 0;
  double z = 0;
  double w = 0;

  Vector4(this.x, this.y, this.z, this.w);
}

class RefLink {
  RefLink? next;
  RefLink(this.next);
}

File: dotnet_core/src/Poco.cs
class Vector4 {
    private double _x;
    private double _y;
    private double _z;
    private double _w;

    public Vector4(double x, double y, double z, double w) {
        this._x = x;
        this._y = y;
        this._z = z;
        this._w = w;
    }
}

class RefLink {
    public RefLink next;
    public RefLink(RefLink next) {
        this.next = next;
    }
}